inkscape/svg2shenzhen/export.py

```diff
@@ -628,32 +628,31 @@ def exportEdgeCut(self, kicad_mod=False):
 
             layer_trans = layer.get('transform')
             if layer_trans:
-                layer_m = simpletransform.parseTransform(layer_trans)
+                layer_m = Transform(layer_trans).matrix
             else:
                 layer_m = IDENTITY_MATRIX
 
             nodePath = ('//svg:g[@inkscape:groupmode="layer"][%d]/descendant::svg:path') % i
             for node in self.document.getroot().xpath(nodePath, namespaces=inkex.NSS):
                 d = node.get('d')
-                p = simplepath.parsePath(d)
+                p = inkex.Path(d).to_arrays()
```

Comment:
This is just `node.path.to_arrays()` no need to convert it yourself.

Reply:
Thanks for the tip. Done!